development/cli/getting_started.rst

```diff
@@ -72,3 +72,85 @@ Common options that can be used with any of phpBB's CLI commands.
    --no-ansi | Disable ANSI (colors) output
    --no-interaction (-n) | Do not ask any interactive question
    --safe-mode | Run in Safe Mode (without extensions)
+
+Install phpBB using the CLI
+===========================
+
+You need to use the ``install/phpbbcli.php``. Open the file ``docs/install-config.sample.yml`` and copy its content in a new file ``install/install-config.yml``. Change the parameters to your needs. For example a mysql database with ``mysqli`` interface, ``localhost`` and with an database user ``bertie`` with the password ``bertiepasswd`` into the database named ``bertiedb``:
+
+.. code-block:: console
+
+        installer:
+            admin:
+                name: admin
+                password: mypassword
+                email: admin@example.org
+
+            board:
+                lang: en
+                name: My Board
+                description: My amazing new phpBB board
+
+            database:
+                dbms: mysqli
+                dbhost: ~
+                dbport: ~
+                dbuser: bertie
+                dbpasswd: bertiepasswd
+                dbname: bertiedb
+                table_prefix: phpbb_
+
+            email:
+                enabled: false
+                smtp_delivery : ~
+                smtp_host: ~
+                smtp_port: ~
+                smtp_auth: ~
+                smtp_user: ~
+                smtp_pass: ~
+
+            server:
+                cookie_secure: false
+                server_protocol: http://
+                force_server_vars: false
+                server_name: localhost
+                server_port: 80
+                script_path: /
+
+            extensions: ['phpbb/viglink']
```

Comment:
Uhm, I'm not sure, but this never worked, by default all extensions are installed.
Maybe I'm wrong

Reply:
That is what my experience was today, too. But I removed this line completely and the installer activated every extension, which were available.

Reply:
I'd test if `extensions: false` or `extensions: ['phpbb/viglink', 'vendor1/ext1', 'vendor2/ext2']` works as intended.